web/js/codeworld_shared.js

```diff
@@ -1485,79 +1485,79 @@ function printMessage(type, message) {
     }
   }
 
-  if (type === 'error' || type === 'warning') {
-    if (!window.alreadyReportedErrors.has(scrubError(message))) {
-      const $reportLink = $('<a>');
-      $reportLink.attr('href', '#');
-      $reportLink.addClass('report-unhelpful');
-      $reportLink.on('click', (event) =>
-        sendUnhelpfulReport(event, message, $reportLink)
-      );
-      $reportLink.text('Not helpful?');
-
-      const $lastOutputBlockMessageContent = $outputBlock
-        .children()
-        .last()
-        .find('.message-content');
-
-      if ($lastOutputBlockMessageContent.children().length) {
-        $lastOutputBlockMessageContent
-          .find('.message-summary')
-          .append($reportLink);
-      } else {
-        $lastOutputBlockMessageContent.append($reportLink);
-      }
-    }
-  }
+  // if (type === 'error' || type === 'warning') {
+  //   if (!window.alreadyReportedErrors.has(scrubError(message))) {
+  //     const $reportLink = $('<a>');
+  //     $reportLink.attr('href', '#');
+  //     $reportLink.addClass('report-unhelpful');
+  //     $reportLink.on('click', (event) =>
+  //       sendUnhelpfulReport(event, message, $reportLink)
+  //     );
+  //     $reportLink.text('Not helpful?');
+
+  //     const $lastOutputBlockMessageContent = $outputBlock
+  //       .children()
+  //       .last()
+  //       .find('.message-content');
+
+  //     if ($lastOutputBlockMessageContent.children().length) {
+  //       $lastOutputBlockMessageContent
+  //         .find('.message-summary')
+  //         .append($reportLink);
+  //     } else {
+  //       $lastOutputBlockMessageContent.append($reportLink);
+  //     }
+  //   }
+  // }
```

Comment:
Would a block comment be better here to reduce diff noise?

Reply:
@nimec01

Reply:
I changed it